### PR TITLE
fix(UI): quick find

### PR DESCRIPTION
**Pull request by homelessbirds**

### CODE REVIEW

**Review by nbaenam — COMMENTED** [review]
Although the `accounts_screenshot-crop_search_1.webp` has been changed, the build displays the old one (`accounts_screenshot-crop_search.webp`).

File: src/content/docs/new-relic-solutions/new-relic-one/introduction-new-relic-platform.mdx [hunk 1]
```diff
@@ -385,9 +386,13 @@ Some notes about sharing:
...
 * Some sharing options have associated time ranges, which may impact later viewings of it. For example, if you use a chart's **Get chart link** option and that chart is set to 'Last 30 minutes', when viewed it will show the last 30 minutes, not the time range displayed when it was shared. To share a specific time range, you must select that time range in the UI.
 
-## Search accounts and entities [#search]
+## Search accounts, capabilities and entities [#search]
+
+To search across all your New Relic data, click **Quick Find** at the top of the New Relic UI.  From here you will be able to search your entities as well as New Relic capabilities. 
+
+Let's say you're a developer for an e-commerce store and you're looking for the service that monitors your cart. You can type `Checkout-service` into the **Search and Navigate** field. You'll be able to see all the entities that match that description. Each result will have an alert severity icon so you can quickly see if there are any issues.
```
Comment (akristen) [inline src/content/docs/new-relic-solutions/new-relic-one/introduction-new-relic-platform.mdx:393]: e-commerce --> ecommerce (not hyphen)

File: src/content/docs/new-relic-solutions/new-relic-one/introduction-new-relic-platform.mdx [hunk 2]
```diff
@@ -431,7 +436,7 @@ You can use keyboard shortcuts to navigate through the UI efficiently.
                 Open global search
             </td>
             <td>
-                `/`
+                Press `K` while holding `Ctrl` + `Cmd`
```
Comment (akristen) [inline src/content/docs/new-relic-solutions/new-relic-one/introduction-new-relic-platform.mdx:439]: Is there a windows equivalent? Looks like some of these alternate between only giving the MacOS shortcuts, only the Windows, or both.

Comment (akristen) [reply src/content/docs/new-relic-solutions/new-relic-one/introduction-new-relic-platform.mdx:439]: Windows equivalent == Ctrl + K (source: big doc you linked!)

**Review by akristen — APPROVED** [review]
Approving, but left some comments! I can re-approve if you want to get all the feedback in now, or you can do a fast follow in a separate PR and I can look at that too!

**Pull request merged**